schema-guide.md

```diff
@@ -1224,59 +1224,199 @@ authors:
 
 ### `definitions.identifier`
 
-- **type**: One of the following `object`s:
-    1. DOI
-        - `type`:
-            - **type**: `enum` with singular value `doi`
-            - **required**: `true`
-            - **description**: The type of identifier.
-        - `value`:
-            - **type**: [`definitions.doi`](#definitionsdoi)
-            - **required**: `true`
-            - **description**: The value of the DOI, e.g. `10.5281/zenodo.1003149`
-        - `description`:
-            - **type**: [`definitions.identifier-description`](#definitionsidentifier-description)
-            - **required**: `false`
-            - **description**: The description of the DOI, e.g. `This is the DOI for version 0.11.4.`
-    1. URL
-        - `type`:
-            - **type**: `enum` with singular value `url`
-            - **required**: `true`
-            - **description**: The type of identifier.
-        - `value`:
-            - **type**: [`definitions.url`](#definitionsurl)
-            - **required**: `true`
-            - **description**: The value of the URL, e.g. `https://github.com/citation-file-format/citation-file-format`.
-        - `description`:
-            - **type**: [`definitions.identifier-description`](#definitionsidentifier-description)
-            - **required**: `false`
-            - **description**: The description of the URL, e.g. `The homepage for the project`.
-    1. Software Heritage identifier
-        - `type`:
-            - **type**: `enum` with singular value `swh`
-            - **required**: `true`
-            - **description**: The type of identifier.
-        - `value`:
-            - **type**: [`definitions.swh-identifier`](#definitionsswh-identifier)
-            - **required**: `true`
-            - **description**: The value of the Software Heritage identifier, e.g. `swh:1:dir:bc286860f423ea7ced246ba7458eef4b4541cf2d`.
-        - `description`:
-            - **type**: [`definitions.identifier-description`](#definitionsidentifier-description)
-            - **required**: `false`
-            - **description**: The description of the Software Heritage identifier, e.g. `The directory object of the repository as stored on Software Heritage.`
-    1. Other
-        - `type`:
-            - **type**: `enum` with singular value `other`
-            - **required**: `true`
-            - **description**: The type of identifier.
-        - `value`:
-            - **type**: Nonempty `string`
-            - **required**: `true`
-            - **description**: The value of the identifier, e.g. `arXiv:2103.06681`
-        - `description`:
-            - **type**: [`definitions.identifier-description`](#definitionsidentifier-description)
-            - **required**: `false`
-            - **description**: The description of the identifier, e.g. `The ArXiv preprint of the paper`.
+- **type**: One of the following `object`s (click to expand/collapse):<br><br>
```

Comment:
```suggestion
- **type**: One of the following `object` types (click to expand/collapse):<br><br>
```

Minor suggestion. I had removed any `\`s ` instances before just because I hate the rendering of the un-\<pre\>-d trailing _s_.